distro/air-quality-distro.conf

```diff
@@ -0,0 +1,23 @@
+require conf/distro/poky.conf
+
+DISTRO = "air-quality-distro"
+
+DISTRO_NAME = "Distro for mini air quality station"
+DISTRO_VERSION = "0.1.0"
+#SDK_VENDOR = "-airqualitysdk"
+
+MAINTAINER = "Wiktor Kawka asinesak353@gmail.com"
+
+#TARGET_VENDOR = "-air-quality"
+
+DISTRO_FEATURES = "${DISTRO_FEATURES_DEFAULT} systemd"
+DISTRO_FEATURES:remove = " alsa bluetooth nfc sysvinit"
+
+# explictely use systemd and disable sysvinit completely
+VIRTUAL-RUNTIME_init_manager = "procd"
```

Comment:
Rm? Overwritten anyways.

Reply:
@punfil I don't understand what do you mean here

Reply:
```
# explictely use systemd and disable sysvinit completely
VIRTUAL-RUNTIME_init_manager = "procd"
# explictely use systemd and disable sysvinit completely
VIRTUAL-RUNTIME_init_manager = "systemd"
VIRTUAL-RUNTIME_initscripts = ""
VIRTUAL-RUNTIME_syslog = ""
```

You declare two different init_managers. I guess systemd is the correct one.

Reply:
Done